config/environments/production.rb

```diff
@@ -4,6 +4,11 @@
   # Code is not reloaded between requests.
   config.cache_classes = true
 
+  config.paperclip_defaults = {
+    :storage => :s3,
+    :bucket => 'flashcardsyuri2349'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.